src/Stripe.Tests.XUnit/topups/_fixture.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Stripe.Tests.Xunit
+{
+    public class topups_fixture
+    {
+        public StripeTopupCreateOptions TopupCreateOptions { get; }
+        public StripeTopupUpdateOptions TopupUpdateOptions { get; }
+        public StripeTopupListOptions TopupListOptions { get; }
+
+        public StripeTopup Topup { get; }
+        public StripeTopup TopupUpdated { get; }
+        public StripeTopup TopupRetrieved { get; }
+        public StripeList<StripeTopup> TopupList { get; }
+
+        public topups_fixture()
+        {
+            StripeSource source = new StripeSourceService(Cache.ApiKey).Create(new StripeSourceCreateOptions
+            {
+                Type = StripeSourceType.AchCreditTransfer,
+                Currency = "usd",
+                Owner = new StripeSourceOwner
+                {
+                    Email = "amount_4242@example.com"
+                }
+            });
+
+            // Sleep for 5 seconds to ensure the Source is chargeable
+            // 1 or 2 seconds are unfortunately not enough.
+            System.Threading.Thread.Sleep(5000);
```

Comment:
This is awful, but I can't see another way for now 😢

Reply:
Fuuu :/ Yeah, I don't know of one either.